src/components/Articlelist.jsx

```diff
@@ -23,12 +24,12 @@ export const Articlelist = () => {
         <div>
             {articles.map((article) => {
                 return (
-                    <div>
-                    <div className="Article-Card">
-                        <h1>{article.title}</h1>
-                        <p>{article.body}</p>
-                    </div>
-                    <br />
+                    <div key={article.article_id}>
+                        <div className="Article-Card">
+                            <Link to={`/Articles/${article.article_id}`}><h1>{article.title}</h1></Link>
+                            <p>{article.body}</p>
```

Comment:
This should not be here. While BE was a while ago, the body should not be returned on the whole article list. Else we will be rendering the entire article making clicking through to read it redundant, and a lot of information to scroll past when using the site.